<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QLabel" name="zahl2">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>100</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="zahl1">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>100</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="rechenzeichen">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>100</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_gleichzeichen">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>100</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>=</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="eingabe">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>100</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QPushButton" name="pruefen">
      <property name="font">
       <font>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Prüfen</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="aufloesen">
      <property name="font">
       <font>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Auflösen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1068</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuZahlenraum">
    <property name="title">
     <string>Zahlenraum</string>
    </property>
    <addaction name="action0_bis_10"/>
    <addaction name="action0_bis_20"/>
    <addaction name="action0_bis_30"/>
    <addaction name="action0_bis_100"/>
    <addaction name="action_m10_bis_p10"/>
    <addaction name="action_m20_bis_p20"/>
    <addaction name="action_m30_bis_p30"/>
    <addaction name="action_m100_bis_p100"/>
   </widget>
   <widget class="QMenu" name="menuOperation">
    <property name="title">
     <string>Operation</string>
    </property>
    <addaction name="actionPlus"/>
    <addaction name="actionMinus"/>
    <addaction name="actionMal"/>
    <addaction name="actionDurch"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="action_ueber"/>
    <addaction name="actionAufloesen"/>
   </widget>
   <addaction name="menuZahlenraum"/>
   <addaction name="menuOperation"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action0_bis_10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0 bis 10</string>
   </property>
  </action>
  <action name="action0_bis_20">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>0 bis 20</string>
   </property>
  </action>
  <action name="action0_bis_30">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0 bis 30</string>
   </property>
  </action>
  <action name="actionPlus">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Plus</string>
   </property>
  </action>
  <action name="actionMinus">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Minus</string>
   </property>
  </action>
  <action name="actionDurch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Durch</string>
   </property>
  </action>
  <action name="actionMal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mal</string>
   </property>
  </action>
  <action name="action_ueber">
   <property name="text">
    <string>Über den Rechentrainer</string>
   </property>
  </action>
  <action name="action_m10_bis_p10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>-10 bis +10</string>
   </property>
  </action>
  <action name="action_m20_bis_p20">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>-20 bis +20</string>
   </property>
  </action>
  <action name="action_m30_bis_p30">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>-30 bis +30</string>
   </property>
  </action>
  <action name="action_m100_bis_p100">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>-100 bis +100</string>
   </property>
  </action>
  <action name="actionAufloesen">
   <property name="text">
    <string>Auflösen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
  </action>
  <action name="action0_bis_100">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0 bis 100</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
